pkg/database/mariadb.go

```diff
@@ -59,6 +68,16 @@ func (m *MariaDB) CreateFullBackup(path string) (bp LogPosition, err error) {
 	if path == "" {
 		return bp, fmt.Errorf("no path given")
 	}
+	defer func() {
+		err = m.toggleSlowQueryLog(slowQeryLogON)
+		log.Error(fmt.Errorf("error enabling slow_query_log: %s", err.Error()))
+	}()
+
+	if err = m.toggleSlowQueryLog(slowQeryLogOFF); err != nil {
+		log.Error(fmt.Errorf("error disabling slow_query_log: %s", err.Error()))
```

Comment:
it uses fmt.Sprint, so no real formatting with placeholders!

Reply:
Bad wording. What I meant was passing the strings directly, since they are only concatenated.

With formatting I would suggest: 
```suggestion
		log.Errorf("error disabling slow_query_log: %s", err.Error())
```